sjb/config/test_cases/test_pull_request_origin_extended_image_registry.yml

```diff
@@ -10,5 +10,25 @@ extensions:
       title: "run extended tests"
       repository: "origin"
       script: |-
+        for pth in ${OS_ROOT:-} "$(pwd)" "$(dirname ${BASH_SOURCE})" "/data/src/github.com/openshift/origin"; do
```

Comment:
In the test env you can be certain Origin is at `/data/src/github.com/openshift/origin`